### ISSUE TITLE
Mailing List

### ISSUE THREAD

**Issue by migueldvb** [lime-rt/lime#175]
Apart from the issue tracker on Github is there any public mailing list to discuss the LIME code development, or to get help with some problem,  discuss a feature etc?

**Comment by smaret** [comment]
There used to be a forum, but I don't think it's still active. We could perhaps open a chat room on [gitter](https://gitter.im/) for this?

**Comment by migueldvb** [comment]
The lime forum used to be here http://brinch.eu/limeforum/

**Comment by christianbrinch** [comment]
That is right. I killed it years ago. Is there any interest in bringing it back?
